File: TI2.4_EUREKABANK_BDDNUBE_REST_DOTNET_GR01/04. CLIWEB/CLIWEB_EurekaBank_RESTDOTNET/src/main/java/ec/edu/gr03/model/Cuenta.java
package ec.edu.gr03.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.io.Serializable;

public class Cuenta implements Serializable {
    
    @JsonProperty("NumeroCuenta")
    private String numeroCuenta;
    
    @JsonProperty("NombreCliente")
    private String nombreCliente;
    
    @JsonProperty("Saldo")
    private Double saldo;
    
    @JsonProperty("Moneda")
    private String moneda;
    
    @JsonProperty("Estado")
    private String estado;
    
    public Cuenta() {}
    
    // Getters y Setters
    
    public String getNumeroCuenta() {
        return numeroCuenta;
    }
    
    public void setNumeroCuenta(String numeroCuenta) {
        this.numeroCuenta = numeroCuenta;
    }
    
    public String getNombreCliente() {
        return nombreCliente;
    }
    
    public void setNombreCliente(String nombreCliente) {
        this.nombreCliente = nombreCliente;
    }
    
    public Double getSaldo() {
        return saldo;
    }
    
    public void setSaldo(Double saldo) {
        this.saldo = saldo;
    }
    
    public String getMoneda() {
        return moneda;
    }
    
    public void setMoneda(String moneda) {
        this.moneda = moneda;
    }
    
    public String getEstado() {
        return estado;
    }
    
    public void setEstado(String estado) {
        this.estado = estado;
    }
}

File: TI2.4_EUREKABANK_BDDNUBE_REST_DOTNET_GR01/04. SERVIDOR/SERV_EurekaBank_RESTDOTNET/ec.edu.pinza.modelo/Cuenta.cs
using System;

namespace ec.edu.pinza.modelo
{
    public class Cuenta
    {
        public string NumeroCuenta { get; set; }
        public string NombreCliente { get; set; }
        public double Saldo { get; set; }
        public string Moneda { get; set; }
        public string Estado { get; set; }

        public Cuenta() { }

        public Cuenta(string numeroCuenta, string nombreCliente, double saldo, string moneda, string estado)
        {
            NumeroCuenta = numeroCuenta;
            NombreCliente = nombreCliente;
            Saldo = saldo;
            Moneda = moneda;
            Estado = estado;
        }
    }
}
